### PR TITLE
Feature/#331 restyle delete unenroll

### PR DESCRIPTION
**Pull request by WorldofJARcraft**
# Resolves #number

This PR changes / adds / introduces the feature of ....

Acceptance Criteria:
- [ ] “Destructive Action Buttons” (Delete Course & Leave Lecture) should all look like the "Delete File"-Button in the Materials Section.

Testing steps:
- search for buttons that represent "dangerous" activities like leaving a course or deleting something
- they should look like the "Leave lecture" button:
image

Steps needed for migration:
- None

---

Definition Of Done:
- [ ] [Class-Diagram](https://www.lucidchart.com/invitations/accept/705a122b-58a9-4dfa-8b16-c936bc0a46bc) updated

### CODE REVIEW

File: app/views/courses/show.html.erb [hunk 1]
```diff
@@ -47,9 +47,9 @@
 <% end %>
 
 <div class="d-flex flex-row justify-content-between mb-2">
-  <%= link_to (fa_icon 'arrow-left', text: 'Back to Overview'), root_path, class: "btn btn-outline-primary" %>
```
Comment (essal0815) [inline app/views/courses/show.html.erb]: This should be unchanged

File: app/views/home/index.html.erb [hunk 2]
```diff
@@ -104,7 +104,7 @@
               <td>
                 <span class="overflowing"><%= course.description %></span>
               </td>
-              <td class="text-right"><%= link_to "Join", join_course_path(id: course.id), method: :post, class: "btn btn-primary" %></td>
+              <td class="text-right"><%= link_to "Join", join_course_path(id: course.id), method: :post, class: "btn btn-outline-danger" %></td>
```
Comment (essal0815) [inline app/views/home/index.html.erb]: This should be unchanged join is no danger

File: app/views/courses/show.html.erb (hunk 1 above)
Comment (WorldofJARcraft) [reply app/views/courses/show.html.erb]: Loooked strange, so I changed it.

File: app/views/home/index.html.erb (hunk 2 above)
Comment (WorldofJARcraft) [reply app/views/home/index.html.erb]: Correct.

**Pull request merged**